api/views/__init__.py

```diff
@@ -302,7 +291,6 @@ def get_queryset(self):
         return flags
 
 
-@swagger_auto_schema(methods=["post"], request_body=WebcompatIssueSerializer)
```

Comment:
@say-yawn I didn't see a reason to keep this? the docs generated by this new `drf-spectacular` library still look fine ...

new swagger:
![image](https://github.com/mozilla/fx-private-relay/assets/71928/0cf8a20d-b316-4ec2-a610-0b74ae78da3d)

new redocly:
![image](https://github.com/mozilla/fx-private-relay/assets/71928/a97810f5-35c3-4047-95b3-6fece5d41544)

Reply:
The new docs are wrong, they do not say what should be in the `POST` body to `report_webcompat_issue`. 

The current docs show the expected JSON in the `POST` body, because of `WebcompatIssueSerializer`:

![report_webcompat_isse](https://github.com/mozilla/fx-private-relay/assets/286017/a921da89-ba19-4954-891e-07db487b7d3e)